sympy/physics/mechanics/joint.py

```diff
@@ -272,9 +336,9 @@ class PinJoint(Joint):
         The parent body of joint.
     child : Body
         The child body of joint.
-    coordinates: List of dynamicsymbols, optional
+    coordinates: dynamicsymbol, optional
         Generalized coordinates of the joint.
-    speeds : List of dynamicsymbols, optional
+    speeds : dynamicsymbol, optional
```

Comment:
Future joints will require a list. I think we should return lists of the symbols for consistency with the future joints.

Reply:
`PinJoint` and `PrismaticJoint` takes `speeds` and `coordinates` as dynamicsymbols because only one argument is required, but returns them as list.

Reply:
Understood. This is the input parameter list. I thought it was the attribute list.